docs/markup-language/README.md

```diff
@@ -933,6 +933,36 @@ The amount of federal income tax that must be withheld from the Employee's
 weekly wages is **$[[Amount of Income Tax Withheld]]**.
 ```
 
+## Clauses
+
+The `Clause` type in OpenLaw is equivalent to an embedded template. It is used to insert commonly used legal logic (boilerplate) - whether simple or complex - into existing templates.
+
+For example, assuming you have created a new template called tester, you could reference it in a second template as below:
+
+```
+[[var]]
+[[var 2]]
+[[clause:Clause(name: "tester"; parameters: sub var -> var)]]
```

Comment:
It's not entirely clear to me what `parameters: sub var -> var` is used for in this line. I think it's mapping parameters? Maybe would be helpful if we show what the contents of the `tester` template are to begin with. Also, it might be helpful to show what the end result of this template looks with the embedded clause in the draft view.

I'm assuming this wouldn't result in a compile error, but can we change `clause` here to distinguish between the `Clause` type? I think that's just a variable name we can change?

And just tiny fix to put a space between `:Clause` to make that consistent with how we're defining variables in the rest of the file (assuming it wouldn't cause a compile error).